components/script/dom/htmlscriptelement.rs

```diff
@@ -338,17 +342,27 @@ fn finish_fetching_a_classic_script(
     load: ScriptResult,
     can_gc: CanGc,
 ) {
-    // Step 11, Asynchronously complete this algorithm with script,
+    // Step 33, Asynchronously complete this algorithm with script,
```

Comment:
The comment above this function says it implements https://html.spec.whatwg.org/multipage/#fetch-a-classic-script, but that this comment belongs to https://html.spec.whatwg.org/multipage/scripting.html#prepare-the-script-element.

Reply:
Also, one small nit, our usual convention is to put a dot after the step number, not a comma.